app/models/job.rb

```diff
@@ -20,7 +20,7 @@ def self.valid_status?(status)
   end
 
   def self.non_deploy
-    includes(:deploy).where(deploys: { id: nil })
+    joins('left join deploys on deploys.job_id = jobs.id').where(deploys: { id: nil })
```

Comment:
making this cheaper ... the include triggered join fetches all the columns, which we don't need